site/layouts/shortcodes/note.html

```diff
@@ -1,6 +1,6 @@
 <aside class="note">
-    <div class="note-icon">
-        {{partial "svg/exclamation.svg" (dict "size" "20px" ) }}
+    <div class="note-content">
+        <strong>{{ .Get 0 }}</strong><br>
+        {{ .Inner | markdownify }}
```

Comment:
This is needed because of breaking changes in https://gohugo.io/news/0.55.0-relnotes/